recipe/meta.yaml

```diff
@@ -1,40 +1,52 @@
 {% set name = "doxygen" %}
 {% set version = "1.8.13" %}
-{% set sha256 = "af667887bd7a87dc0dbf9ac8d86c96b552dfb8ca9c790ed1cbffaa6131573f6b" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.src.tar.gz
-  url: ftp://ftp.stack.nl/pub/users/dimitri/{{ name }}-{{ version }}.src.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - doxygen_test_patch.diff
+  fn: {{ name }}-{{ version }}.src.tar.gz                                                  # [not win]
+  url: ftp://ftp.stack.nl/pub/users/dimitri/{{ name }}-{{ version }}.src.tar.gz            # [not win]
+  sha256: af667887bd7a87dc0dbf9ac8d86c96b552dfb8ca9c790ed1cbffaa6131573f6b                 # [not win]
+
+  fn: {{ name }}-{{ version }}.windows.bin.zip                                             # [win32]
+  url: http://ftp.stack.nl/pub/users/dimitri/{{ name }}-{{ version }}.windows.bin.zip      # [win32]
+  sha256: 215800d19d0e77fa400694af74685bbd3000463f0d7f18b6c7b743d2d6a1422f                 # [win32]
+
+  fn: {{ name }}-{{ version }}.windows.x64.bin.zip                                         # [win64]
+  url: http://ftp.stack.nl/pub/users/dimitri/{{ name }}-{{ version }}.windows.x64.bin.zip  # [win64]
+  sha256: c59e92ab9f82e156723dd3efb9433948dcb84facc21d6efb50f4d6c2d7aa6c5c                 # [win64]
+
+  patches:                                                                       # [not win]
+    - doxygen_test_patch.diff                                                    # [not win]
     # CMake finds iconv_open in glibc, but finds iconv.h from libiconv.
     # Make sure both are found from one place.
-    - find_iconv.patch
-
+    - find_iconv.patch                                                           # [not win]
+  
 build:
   number: 0
-  skip: True  # [win or not py35]
-  features:
-      - vc9    # [win and py27]
-      - vc10   # [win and py34]
-      - vc14   # [win and py35]
+  skip: True  # [not py35]
+  script:                                                  # [win]
+      - mkdir %SCRIPTS%                                    # [win]
+      - copy doxygen.exe %SCRIPTS%\ || goto :eof           # [win]
+      - copy doxyindexer.exe %SCRIPTS%\ || goto :eof       # [win]
+      - copy doxysearch.cgi.exe %SCRIPTS%\ || goto :eof    # [win]
+      - copy libclang.dll %LIBRARY_BIN%\ || goto :eof      # [win]
+      - copy libclang.dll %LIBRARY_BIN%\ || goto :eof      # [win]
```

Comment:
@isuruf It's easy to build it on Windows, but _without clang support_. There are no precompiled _static_ version of `libclang` on Windows.

Unfortunately, Doxygen depends on clang for "a more accurate source browsing experience, and better syntax highlighting, cross referencing, and call graphs" [(ref.)](http://lists.llvm.org/pipermail/cfe-dev/2013-May/029732.html).

I agree this could be a problem, but I think there are no other packages using `libclang.dll`, and there are no `libclang` packages for Windows. Whenever `libclang` or `clang` is added to conda-forge, we can update this recipe to depend on it. What do you think?